settings/DevHome.Settings/Views/ExperimentalFeaturesPage.xaml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Page
+    x:Class="DevHome.Settings.Views.ExperimentalFeaturesPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:behaviors="using:DevHome.Common.Behaviors"
+    xmlns:converters="using:CommunityToolkit.WinUI.Converters"
+    xmlns:vm="using:DevHome.Settings.ViewModels"
+    behaviors:NavigationViewHeaderBehavior.HeaderMode="Never">
+    <Page.Resources>
+        <converters:DoubleToVisibilityConverter x:Key="CountToVisibilityConverter" GreaterThan="0" FalseValue="Visible" TrueValue="Collapsed"/>
+    </Page.Resources>
+    <Grid
+        MaxWidth="{ThemeResource MaxPageContentWidth}"
+        Margin="{ThemeResource ContentPageMargin}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+
+        <BreadcrumbBar
+            x:Name="BreadcrumbBar"
+            Margin="0,0,0,16"
+            ItemClicked="BreadcrumbBar_ItemClicked"
+            ItemsSource="{x:Bind Breadcrumbs}" />
+
+        <ScrollViewer Grid.Row="1" VerticalAlignment="Top">
+            <StackPanel Orientation="Vertical">
+                <ItemsRepeater ItemsSource="{x:Bind ViewModel.Features}">
+                    <ItemsRepeater.ItemTemplate>
+                        <DataTemplate x:DataType="vm:ExperimentalFeature">
+                            <Grid ColumnDefinitions="*,Auto"  Padding="4" Margin="4">
```

Comment:
Instead of Margin=4, use `SettingsCardSpacing` for consistency (ideally between SettingsCards :-) )